resources/hpxml-measures/BuildResidentialHPXML/measure.rb

```diff
@@ -2883,6 +2883,23 @@ def arguments(model)
     arg.setDefaultValue(false)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('electricity_co_2_emissions_filepaths', false)
+    arg.setDisplayName('Electricity CO2 Emissions: CSV Paths')
+    arg.setDescription('Absolute/relative paths (comma-separated) of electricity CO2 emissions factor schedule file with hourly values.')
+    args << arg
+
+    electricity_co2_emissions_units_choices = OpenStudio::StringVector.new
+    electricity_co2_emissions_units_choices << HPXML::EmissionsScenario::UnitsKgPerMWh
+    electricity_co2_emissions_units_choices << HPXML::EmissionsScenario::UnitsKgPerMBtu
+    electricity_co2_emissions_units_choices << HPXML::EmissionsScenario::UnitsLbPerMWh
+    electricity_co2_emissions_units_choices << HPXML::EmissionsScenario::UnitsLbPerMBtu
+
+    arg = OpenStudio::Measure::OSArgument.makeChoiceArgument('electricity_co_2_emissions_units', electricity_co2_emissions_units_choices, false)
+    arg.setDisplayName('Electricity CO2 Emissions: CSV Paths')
+    arg.setDescription('Electricity CO2 emissions factor units.')
+    arg.setDefaultValue(HPXML::EmissionsScenario::UnitsKgPerMWh)
+    args << arg
+
```

Comment:
@shorowit Can you help me re-think these arguments based on our discussion around having name/filepath pairs to define scenarios? Should we define a max number of scenarios (similar to heating systems, mechanical ventilation systems, PV systems, etc)? Or perhaps, like the `filepaths` argument, have a corresponding comma-separated `names` argument?

Reply:
Comma-separated `names` argument works for me. Also note that for electricity, only the MWh units apply (the MBtu units only apply to fossil fuels), so you should remove the two MBtu options.